.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
`package-lock.json` was already untracked, but would still be created. This avoids it being created in the first place. (Having the file around can mask errors, as what's installed locally only may be different from what's installed by the CI and end users.) (Also, `package-lock.json` is not publishable in the npm registry, so committing it is not a good idea, and neither `package-lock.json` nor `npm-shrinkwrap.json` are recommended for libraries.)